app/assets/javascripts/discourse/app/lib/plugin-api.js

```diff
@@ -2298,6 +2299,27 @@ class PluginApi {
     addSidebarPanel(func);
   }
 
+  /**
+   * EXPERIMENTAL. Do not use.
+   * Support for adding links to specific admin sidebar sections.
+   *
+   * This is intended to replace the admin-menu plugin outlet from
+   * the old admin horizontal nav.
+   *
+   * ```
+   * api.addAdminSidebarSectionLink("root", {
+   *   name: "unique_link_name",
+   *   route: "emberRouteId",
+   *   label: "admin.some.i18n.label.key",
+   *   icon: "icon-name",
+   *   href: "(optional) can be used instead of the route",
```

Comment:
I think documenting the properties of the object using JSDoc will make the API easier to understand. This is what I did for the `addCommunitySidebarSectionLink` API

https://github.com/discourse/discourse/blob/18f8ebb2eb30ae50ff911bbb1411cc7c2581d63f/app/assets/javascripts/discourse/app/lib/plugin-api.js#L2101-L2108

Reply:
Yes will do :+1: